README.md

```diff
@@ -74,12 +74,12 @@ The `refract` variable above has the following JSON value.
 }
 ```
 
-### Converting from Types
+### Converting from Elements
 
-If the JSON above is used, it can be converted back to Minim types to make a roundtrip.
+If the JSON above is used, it can be converted back to Minim elements to make a roundtrip.
 
 ```javascript
-var arrayType = minim.convertFromDom(aboveJson);
+var arrayElement = minim.convertFromElement(aboveJson);
```

Comment:
In this method the word `element` seems to refer to a JSON serialization of refract, while above `convertToElement` uses the word `element` to mean an instance of an element type. I wonder if we can't make this a little more clear / less ambiguous.

```js
var arrayElement1 = minim.fromValue([1, 2, 3]);
var refracted = arrayElement1.toRefract();
var arrayElement2 = minim.fromRefract(refracted);
```